week1/homework/src/server.js

```diff
@@ -1,16 +1,37 @@
 'use strict';
 
 const http = require('http');
+const whatIDO = require('./commands')
 
 /* `createServer` MUST return an instance of `http.Server` otherwise the tests
  * will fail.
  */
-function createServer(port) {
+
+function createServer(PORT) {
   let state = 10;
 
+
   const server = http.createServer((request, response) => {
     // TODO: Write your homework code here
+    const URLString = request.url;
+    const URLArray = URLString.split('/').filter(command => ["state", "add", "subtract", "reset"].includes(command));
+    console.log('Initiating commands')
+
+    for (const command of URLArray) {
+      state = whatIDO(command, state)
+    }
+
+    response.writeHead(200, { 'Content-Type': 'application/json' });
```

Comment:
the header should be 403 if the path is not the specified one.

Reply:
404 is maybe better ?